src/components/HelpBubble.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { SyntheticEvent, useRef, useState } from 'react';
+import ClickableContainer from './ClickableContainer';
+import Icon from './Icon';
+import Popover from './Popover';
+import PopoverBody from './PopoverBody';
+import PopoverHeader from './PopoverHeader';
+
+interface HelpBubbleProps
+  extends Omit<
+    React.ComponentProps<typeof Popover>,
+    'isOpen' | 'toggle' | 'target'
+  > {
+  title?: React.ReactNode;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+let count = 0;
+
+function getID() {
+  return `help-bubble-${count++}`; // eslint-disable-line no-plusplus
+}
+
+const style = {
+  cursor: 'pointer',
+};
+
+function HelpBubble(props: HelpBubbleProps) {
+  const idRef = useRef<string | undefined>();
+  if (!idRef.current) idRef.current = getID();
+  const id = idRef.current;
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = (e: SyntheticEvent) => {
+    e.stopPropagation();
+    setIsOpen((prevIsOpen) => !prevIsOpen);
+  };
+
+  const { title, children, className, ...other } = props;
+
+  return (
+    <span className={className} style={style}>
+      <ClickableContainer aria-label="More Info" tag="span" onClick={toggle}>
```

Comment:
Would it be better to use `<Button color="link">` with the appropriate padding?  ClickableContainer was kind of a workaround for bigger areas like headers, etc.

Reply:
yeah I agree, I would much rather use an actual `button` tag. 

`<Button color="link">` seems to mostly work, but we lose the outline style applied when the help bubble receives focus - not sure how to get that to show up w/o custom styles?


![Screen Shot 2021-05-10 at 5 26 30 PM](https://user-images.githubusercontent.com/11209547/117740277-f229fe00-b1b4-11eb-9947-4f4440d045eb.png)
![Screen Shot 2021-05-10 at 5 28 14 PM](https://user-images.githubusercontent.com/11209547/117740357-1f76ac00-b1b5-11eb-8275-8ed67bc97ee0.png)

Reply:
Hmm, let me see what the delta issue

Reply:
Looks like it's bootstrap removing box-shadow on :focus in favor of text underline:

<img width="309" alt="Screen Shot 2021-05-11 at 1 41 03 PM" src="https://user-images.githubusercontent.com/18536746/117882046-cb280680-b25e-11eb-8fea-2b418bc7d5e4.png">

Perhaps we should use `<Button color="link">` and adjust focus styles in Bootstrap theme?  Given the icon does not show underline